File: loops/fizzbuzz.py
#jasper holt, fizzbuzz python
x=1
while x < 51:
    if x%3==0 and x%5==0:
        print("fizzBuzz")
    elif x%3==0:
        print("fizz")
    elif x%5==0:
        print("Buzz")
    else:
        print(x)
    x+=1 

File: loops/FizzBuzz.c
// Jasper Holt, TEMPLATE
#include <stdio.h>
int x = 1;
int main(void){
    
    while (x < 51){
    if (x%3==0 && x%5==0){
        printf("fizzBuzz\n");
    }else if (x%3==0){
        printf("fizz\n");
    }else if (x%5==0){
        printf("Buzz\n");
    }else{
        printf("%d\n",x);
    }
    x+=1;
 };


  
    return 0;
}